src/main/content/_assets/js/header.js

```diff
@@ -28,4 +26,40 @@ $(document).ready(function() {
         }
     });
 
+    if ($('#whatsNewModal').attr('href')) {
+        $("#modal-body").load($('#whatsNewModal').attr('href'), function(response, status) {
```

Comment:
no need to load it, just have liquid place the what's new content in the modal during build

Reply:
The font matter for our docs doesnt allow us to load and format the file correctly. 
![Screen Shot 2019-08-15 at 9 54 29 AM](https://user-images.githubusercontent.com/39460219/63099241-b2cc7000-bf42-11e9-9df2-3c7e13d561ff.png)

I'm using the same same function thats used to load all our docs for the docs page (.load()) to load in the 'whats-new' doc

Reply:
so instead of this js you could do something like: